### DISCUSSION TITLE
Online Tracking

### DISCUSSION THREAD

**Discussion by taebek** [osmandapp/OsmAnd#18454] [Q&A]
Hello.
Can I get some help with setting up online tracking? How can I, as a simple user, create a website that can connect to the OsmAnd app? What software do I need to do this? Could you give me some information about the possibilities, the exact steps?

**Comment by AndyB69** [comment]
Hello - I've got the same question as @taebek please.

Under:  Settings > {profile} > Plugin Settings - Trip recording > **Online tracking**

Click on the Web Address field and it asks for a website URL - but what URL should be put here? There are no instructions either on screen or in the OsmAnd web documentation that provides any clues. 

I'm a Pro user, can the OsmAnd online map URL be used somehow? i.e. https://www.osmand.net/map/...

**Comment by AndyB69** [comment]
Hello, can someone get back to us on this question, please?

**Comment by tommycrock** [comment]
It looks like you can use a site called travel map to do this, I presume the paid option [1]. Or, someone seems to have written an app that integrates this feature with Google maps [2]. Maybe one of those helps you find the answer?

1. https://travelmap.net/support/live-tracking/osmand
2. https://github.com/jdstemmler/osmand-live-track

**Comment by JasonSanDiego** [comment]
Traccar [1] is a nice open source project that supports the OSMand protocol and can easily be deployed to Proxmox as an LXC container using proxmox helper scripts [2]. There are some detailed docs on OSMand setup for Traccar, such as pointing it at port 5055, not the web port [3]. I ran it over Tailscale successfully, which saved trouble of getting a secure connection.

1. https://www.traccar.org/
2. https://tteck.github.io/Proxmox/
3. https://www.traccar.org/osmand/
